docs/src/tutorials/getting_started/transitioning_from_matlab.jl

```diff
@@ -0,0 +1,280 @@
+# still fairly adventurous, being willing to reformulate a nonlinear objective
+# in terms of conic constraints, for example. JuMP does no such thing: it only
+# reformulates objectives into objectives, and constraints into constraints, and
+# is fairly conservative at that. As a result, you might need to do some
+# reformulations manually, for which a good guide is available [here](@ref conic_tips_and_tricks).
+
+# An interesting possibility that JuMP offers is turning off reformulations
+# altogether, which will reduce latency if they're in fact not needed, or
+# produce an error if they are. To do that, set your optimizer with the option
+# `add_bridges = false`, as for example, in `set_optimizer(model, Hypatia.Optimizer, add_bridges=false)`
+
+# ## Vectorization
+
+# In MATLAB it is absolutely essential to "vectorize" your code to obtain
+# acceptable performance. This is because MATLAB is a very slow interpreted
+# language, which sends your commands to extremely fast libraries. When you
+# "vectorize" your code you're minimizing the MATLAB part of the work and sending
+# it to the libraries instead. There's no such duality with Julia. Everything you
+# write and most libraries you use will compile down to LLVM, so "vectorization"
+# has no effect.
+
+# For example, if you are writing a linear program in MATLAB and instead of the
+# usual `constraints = [v >= 0]` you write:
+# ```matlab
+# for i = 1:n
+#    constraints = [constraints, v(i) >= 0];
+# end
+# ```
+# performance will be horrible. With Julia, on the other hand, there's hardly
+# any difference between `@constraint(model, v >= 0)` and
+
+for i in 1:n
+    @constraint(model, v[i] >= 0)
+end
+
+# ## Rosetta stone
+
+# To finish this tutorial, we show a complete example of the same optimization
+# problem being solved with JuMP, YALMIP, and CVX. It is an SDP computing a
+# simple lower bound on the random robustness of entanglement using the partial
+# transposition criterion. The code is complete apart from the function that does
+# partial transposition. With JuMP we use the function `partialtranspose` from
+# [Convex.jl](https://jump.dev/Convex.jl/stable/). With both YALMIP and CVX we
+# use the function `PartialTranspose` from [QETLAB](https://github.com/nathanieljohnston/QETLAB).
+
+# ### JuMP
+
+using JuMP
+import Convex
+import Hypatia
+import LinearAlgebra
+
+function random_state_pure(d)
+    x = randn(ComplexF64, d)
+    y = x * x'
+    return LinearAlgebra.Hermitian(y / LinearAlgebra.tr(y))
+end
+
+function robustness_jump(d)
+    model = Model()
+    @variable(model, λ)
+    ρ = random_state_pure(d^2)
+	id = LinearAlgebra.Hermitian(LinearAlgebra.I(d^2))
+	ρT = LinearAlgebra.Hermitian(Convex.partialtranspose(ρ, 1,[d ,d]))
+    PPT = @constraint(model, ρT + λ * id in HermitianPSDCone())
+    @objective(model, Min, λ)
+    set_optimizer(model, Hypatia.Optimizer; add_bridges = false)
+    set_attribute(model, "verbose", true)
+    optimize!(model)
```

Comment:
One thing this example doesn't touch on is solver and result statuses. We should either add `@assert is_solved_and_feasible(model)`, or explain how to query statuses in YALMIP and CVX.

Reply:
That's a good point, we should add a section on querying solver status, this is something everyone will need.